tests/spec/fs.chmod.spec.js

```diff
@@ -39,6 +39,20 @@ describe('fs.chmod, fs.fchmod', function() {
     });
   });
 
+  it('Should be a valid mode value to set permissions', function(done){
```

Comment:
Style nit: we tend to use a lower case letter on the test descriptions.  So use `should` vs. `Should`